server/athenian/api/openapi/openapi.yaml

```diff
@@ -1083,6 +1260,90 @@ components:
       items:
         $ref: '#/components/schemas/DeveloperID'
       type: array
+    TeamMember:
```

Comment:
I wonder if we can merge `TeamMember` with `User` which are reported in `/account`...